components/health-status-indicator.tsx

```diff
@@ -123,20 +132,36 @@ const HealthStatusIndicator = ({
       {healthStatus.diagnostics?.ssm && (
         <div className="ml-5 mt-2 text-xs">
           <details className="cursor-pointer">
-            <summary className="text-blue-600 hover:text-blue-800">SSM Debug Info</summary>
+            <summary className="text-blue-600 hover:text-blue-800">Diagnostic Information</summary>
```

Comment:
The nature of the error is such that it's still not appearing in the frontend.

The request is timing out here:
![image](https://github.com/user-attachments/assets/535d9bc1-1924-4a54-b350-aac24b693262)

- can you figure out to not allow my ssm send command to hang for more than 15 seconds? Manipulating the timeout to less than 30 wasn't allowed by AWS
- edit the npm run health check command so that it ends in less than 5 seconds if we detect it lasting that long for whatever reason